### DISCUSSION TITLE
vfd control over rs485

### DISCUSSION THREAD

**Discussion by wbendick** [fra589/grbl-Mega-5X#284] [Q&A] [answered]
Is it possible to control my vfd over rs485 with grbl mega 5x? I saw that it was possible in grblhal, is this feature here too?

**Comment by fra589** [comment, marked as answer]
Hi @wbendick,  
  
To use RS485 with Arduino Mega, you need a converter since the Arduino Mega is not hardware compatible with RS485.
Using an RS485 to RS232 hardware converter, you should be able to use grbl-Mega-5X with RS485.

@++;
Gauthier.
